### PR TITLE
Fix Editions (and some refactoring)

### PR DESCRIPTION
**Pull request by SiAdcock**
Fixes an issue we introduced in #9 whereby links pointed to the Android app repo instead of the Editions repo.

I've also done a bit of refactoring to make the code clearer, and improved the developer experience slightly by:

- ignoring `node_modules`
- upgrading Node to v10
- adding an npm script that kicks of the `builds.gutools` dev server
- updating the README

### CODE REVIEW

File: builds/index.html [hunk 1]
```diff
@@ -60,29 +60,59 @@ <h1 class="text-center">Daily Edition</h1>
           <div class="jumbotron">
             <h1 class="text-center">Editions</h1>
             <p class="lead text-center">Dev version {{ releases.androidEditions.development.version }}<br>
-              <a :href="releases.android.development.teamcity">{{ releases.androidEditions.development.date }}</a> 
-              <a :href="releases.android.development.github">#{{ releases.androidEditions.development.sha1 }}</a></p>
+              <a :href="releases.androidEditions.development.teamcity">{{ releases.androidEditions.development.date }}</a> 
+              <a :href="releases.androidEditions.development.github">#{{ releases.androidEditions.development.sha1 }}</a></p>
```
Comment (SiAdcock) [inline builds/index.html:64]: The bug fix!

**Review by jacobwinch — APPROVED** [review]
Looks great 👍

**Comment by jorgeazevedo** [comment]
Do you know if `reduce`, `fetch` and arrow functions are available on Android 5 and iOS 9 devices? I'm thinking at least arrow functions aren't available in iOS 9, according to #6

**Comment by SiAdcock** [comment]
@jorgeazevedo really good points!

- reverted arrow functions and fancy computed property syntax
- included a `fetch` polyfill as I feel icky about using jQuery just for AJAX sugar
- `reduce` is part of ES5 and [is supported by iOS Safari 9](https://caniuse.com/#feat=es5)
- added a note about browser support to the README

**Comment by jorgeazevedo** [comment]
Thanks for adding that note to the README. It's not written anywhere that this needs to support the same iOS/Android versions that our apps support, sorry about that!

**Review by jorgeazevedo — APPROVED** [review]

**Pull request merged**